app/models/category.rb

```diff
@@ -0,0 +1,7 @@
+class Category < ApplicationRecord
+  belongs_to :user
+  has_many :expenses
```

Comment:
- [ ] [REQUIRED] Please implement previous reviewer's suggestion about add`association` for a `category` belongs to a `user` 🙏🏼

Reply:
Hi. this is implemented as well as the Readme update.